syntax = "proto2";
import "CoreMessages.proto";

package AUX;

message DummyAuxData
{
	optional fixed32  var1   = 1;
	optional string   str1   = 2;
	optional fixed64  var2   = 3;
	optional AnyArray array1 = 4;
}
